.travis.yml

```diff
@@ -69,6 +69,8 @@ install:
   - pip install -U pip
   - pip install git+https://github.com/jssuzanne/sqlalchemy-utils.git@mssql_and_pymssql#egg=sqlalchemy_utils
   - pip install wheel
+  - if [[ $TRAVIS_PYTHON_VERSION == 'pypy3.5' ]]; then pip install psycopg2cffi; else pip install psycopg2-binary; fi
+  - if [[ $SQLPYCLIENT == 'pymssql' ]]; then pip install 'pymssql<3.0'; fi
```

Comment:
why dont modify the $SQLPYCLIENT ?

Reply:
Actually yes it works ! ;) Corrected travis config file to only modify SQLPYCLIENT variable when needed.